macros/PG.pl

```diff
@@ -51,9 +51,9 @@ sub DOCUMENT {
 	$solutionExists        		= $PG->{flags}->{solutionExists};
 	$hintExists            		= $PG->{flags}->{hintExists};
 	$pgComment                  = '';
-	%external_refs         		= %{ $PG->{external_refs}};
+	# %external_refs         		= %{ $PG->{external_refs}};
```

Comment:
And here as well. Perhaps we should have a policy about keeping old code in comments. Maybe the policy would be never do it. Or maybe, you can do it but you need to add commentary about that code. Or something.

Reply:
I agree, except possibly in rare cases. Delete it!